src/main/java/org/spongepowered/api/world/server/ServerWorld.java

```diff
@@ -147,4 +150,32 @@ default UUID getUniqueId() {
 
     @Override
     Collection<ServerPlayer> getPlayers();
+
+    /**
+     * Gets all {@link Raid}s occuring in this {@link ServerWorld}
+     *
+     * <p>Please note by default, some {@link DimensionType}s such as {@link DimensionTypes#THE_NETHER}
+     * will not contain any Raids because the game prevents Raids from starting in the nether.</p>
+     *
+     * @return All the raids in this world.
+     */
+    Collection<Raid> getRaids();
+
+    /**
+     * Gets the {@link Raid} occurring at a position in the world.
+     *
+     * @param blockPosition The location of the Raid.
+     * @return The raid at that location, if present
+     */
+    Optional<Raid> getRaidAt(Vector3i blockPosition);
+
+    /**
+     * Checks if a {@link Raid} is occurring at a position in the world.
+     *
+     * @param blockPosition The location of the Raid.
+     * @return True if a Raid is present at the location.
+     */
+    default boolean hasRaidAt(Vector3i blockPosition) {
```

Comment:
Our plugin devs understand Optional. Not necessary to sugar this.

Reply:
I agree, defaulted simplistic "check if the optional is present" methods are bloat.